docs/API/InstrumentFormat.md

```diff
@@ -196,17 +241,24 @@ save it. The format is as follows:
 
 `Name`: Required. Follows PageElement.Name rules.
 
-`Description`: Required. Follows PageElement.Name rules.
+`Description`: Required. Follows PageElement.Description rules.
 
-`Options.Type`: Either "large" or "small". If "large", the user is meant to enter
+`Options.Type`: If "large", the user is meant to enter
               a lot of text (ie. a comment box) and is likely to be represented
               by a textarea in an HTML implementation.
-              If "small" the user is meant to enter a little text and is likely
+              Else the user is meant to enter a little text and is likely
               to be implemented by a <input type="text"> in an HTML implementation.
-              Default: small
-
+              Default: ""
+```
+Currently if its 'large' we render a textarea, and if it's anything else we render a normal text.
+ZV
```

Comment:
I don't think the spec should change because of that, that's just an implementation detail of how invalid instruments are interpreted.